module1/homework/lengthSort/sort.hpp

```diff
@@ -0,0 +1,7 @@
+#pragma once
+
+#include <deque>
+#include <forward_list>
+#include <string>
+
+std::deque<std::string> lengthSort(std::forward_list<std::string> listToSort);

```

Comment:
listToSort is const

Reply:
No it isn't. Please read the code first.

Reply:
Ok, the implementation caught me off guard.
I don't think we were supposed to modify listToSort, but probably @ziobron or @nauka-programowania-MA should speak up.

Reply:
It's not modified :) Only a copy of the list will be modified because it's taken by a copy.
It meets our requirements. Now we don't care much about performance.